.github/workflows/pr.yml

```diff
@@ -140,6 +144,79 @@ jobs:
       - name: Integration tests
         if: startsWith(matrix.os, 'macos') != true
         run: make test_integration
+  conformance-tests:
+    strategy:
```

Comment:
This looks like a copy of the integration test workflow? Shouldn't this call go test on cmd?

Reply:
There is no `cmd` directory, there are `pulumi-language-dotnet` and `integration_tests`